package.json

```diff
@@ -111,25 +111,25 @@
     "@types/debug": "^4.1.10",
     "@types/eslint-config-prettier": "^6.11.2",
     "@types/mocha": "^10.0.5",
-    "@types/node": "^20.11.0",
+    "@types/node": "^20.12.12",
     "@typescript-eslint/eslint-plugin": "^7.0.1",
-    "@typescript-eslint/parser": "^7.0.1",
+    "@typescript-eslint/parser": "^7.11.0",
```

Comment:
See https://github.com/typescript-eslint/typescript-eslint/issues/8211#issuecomment-2106427365, you will need v8 for this to work.

Reply:
mhm or maybe you can already use Eslint v9 with @typescript-eslint/parser v7?

Reply:
I checked some rules and it seems to work fine, but I'll hold off on merging this until the release, no rush for linting.

Reply:
I just tested a v9 setup in a different project and something I would recommend is add a `/.vscode/settings.json` with:
```
{
  "eslint.experimental.useFlatConfig": true
}
```
So that VSCode picks it up.

Reply:
Yeah, I have this in my VSCode settings already.  I don't think I'll put it in the repo as I'm not a fan of committing editor-specific settings.  Maybe we mention it in the dev docs.  However, it shouldn't be a problem when we get the new version of `vscode-eslint`: https://github.com/microsoft/vscode-eslint/issues/1644